### PR TITLE
Crossed

### PR DESCRIPTION
**Pull request by Cdey78**
Oh shit, it's gonna'  get HOT in here

:cl: Cdey
add: Muh sun, IT BURNS
/:cl:

[why]: # Why not?

### CODE REVIEW

**Comment by Kmc2000** [comment]
Tested?

File: config/admins.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@
 ###############################################################################################
 
 Kmc2000	= Host
-Cdey = Moderator
+Cdey = Game-Admin
```
Comment (Kmc2000) [inline config/admins.txt]: :thinking:

File: StarTrek13/Ships/overmap.dm [hunk 2]
```diff
@@ -108,6 +108,28 @@ var/global/list/global_ship_list = list()
 	max_health = 1000000
 	health = 1000000
 
+/obj/structure/space_object/simulated
+	name = "star"
+	desc = "Flying into this always end up with horrid consequences."
```
Comment (Kmc2000) [inline StarTrek13/Ships/overmap.dm]: Always ends up well

File: config/admins.txt (hunk 1 above)
Comment (Cdey78) [reply config/admins.txt]: :thonk:
had to change it for testing

**Pull request merged**